TypeScript interface augmentation for `SymbolConstructor` conflicts with `symbol-observable`

<!--
Thank you for raising your concerns, we appreciate your feedback and contributions to this repository.

Before you continue, consider the following:

If you have a "How do I do ...?" question, it is better for you and for us that this question is placed in [StackOverflow](http://stackoverflow.com/questions/tagged/rxjs5) or some chat channel. This way, you are making it easier for others to learn from your experiences too.

These "Issues" are meant only for technical problems, bugs, and proposals related to the library.

If your issue is a bug, please follow the format below:
-->

**RxJS version:**

`6.1.0`

**Code to reproduce:**

Include `rxjs@6.1.0` and `symbol-observable@1.2.0` into compilation

**Expected behavior:**

No TypeScript errors

**Actual behavior:**

TypeScript error
```
node_modules/rxjs/internal/symbol/observable.d.ts(4,9): error TS2687: 
    All declarations of 'observable' must have identical modifiers.
```

**Additional information:**

Type definition from `symbol-observable`:

```ts
declare global {
  export interface SymbolConstructor {
    readonly observable: symbol;
  }
}
```

Type definition from `rxjs`:

```ts
declare global {
  interface SymbolConstructor {
    observable: symbol;
  }
}
```

Note the difference in `readonly`.

--- a/src/internal/symbol/observable.ts
+++ b/src/internal/symbol/observable.ts
@@ -5,7 +5,7 @@ import { root } from '../util/root';
   however, we are no longer polyfilling Symbol.observable */
 declare global {
   interface SymbolConstructor {
-    observable: symbol;
+    readonly observable: symbol;
   }
 }
 

